sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs

```diff
@@ -596,105 +714,289 @@ public partial class ProjectEnvironmentTypeResource : Azure.ResourceManager.ArmR
-        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public CatalogSyncState(string value) { throw null; }
+        public static Azure.ResourceManager.DevCenter.Models.CatalogSyncState Canceled { get { throw null; } }
+        public static Azure.ResourceManager.DevCenter.Models.CatalogSyncState Failed { get { throw null; } }
+        public static Azure.ResourceManager.DevCenter.Models.CatalogSyncState InProgress { get { throw null; } }
+        public static Azure.ResourceManager.DevCenter.Models.CatalogSyncState Succeeded { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.DevCenter.Models.CatalogSyncState other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.DevCenter.Models.CatalogSyncState left, Azure.ResourceManager.DevCenter.Models.CatalogSyncState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.DevCenter.Models.CatalogSyncState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.DevCenter.Models.CatalogSyncState left, Azure.ResourceManager.DevCenter.Models.CatalogSyncState right) { throw null; }
+        public override string ToString() { throw null; }
     }
-    public partial class ScheduleData : Azure.ResourceManager.Models.ResourceData
+    public partial class CheckNameAvailabilityContent
     {
-        public ScheduleData() { }
-        public Azure.ResourceManager.DevCenter.Models.ScheduledFrequency? Frequency { get { throw null; } set { } }
-        public string ProvisioningState { get { throw null; } }
-        public Azure.ResourceManager.DevCenter.Models.EnableStatus? State { get { throw null; } set { } }
-        public string Time { get { throw null; } set { } }
-        public string TimeZone { get { throw null; } set { } }
-        public Azure.ResourceManager.DevCenter.Models.ScheduledType? TypePropertiesType { get { throw null; } set { } }
+        public CheckNameAvailabilityContent() { }
+        public string Name { get { throw null; } set { } }
+        public Azure.Core.ResourceType? ResourceType { get { throw null; } set { } }
     }
-    public partial class ScheduleResource : Azure.ResourceManager.ArmResource
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CheckNameAvailabilityReason : System.IEquatable<Azure.ResourceManager.DevCenter.Models.CheckNameAvailabilityReason>
```

Comment:
According to the review guide, this need rename to DevCenterNameUnavailableReason

Reply:
Updated